fe/fe-core/src/main/java/com/starrocks/clone/ColocateTableBalancer.java

```diff
@@ -743,7 +743,7 @@ private boolean checkBackendAvailable(Long backendId, SystemInfoService infoServ
             // 1. BE is dead for a long time
             // 2. BE is under decommission
             if ((!be.isAlive() &&
-                    (currTime - be.getLastUpdateMs()) > Config.tablet_sched_repair_delay_factor_second * 1000 * 2)
+                    (currTime - be.getLastUpdateMs()) > Config.tablet_sched_colocate_be_down_tolerate_time_s * 1000)
```

Comment:
we shouldn't times 1000 here.

Reply:
why not, the config param item is second.

Reply:
yes, you're right